elasticsearch_dsl/function.py

```diff
@@ -137,3 +146,29 @@ class Gauss(ScoreFunction):
 
 class Exp(ScoreFunction):
     name = "exp"
+
+
+class DecayFunction(AttrDict[Any]):
```

Comment:
It seemed overkill to autogenerate just a single function here, so I coded this one by hand using the autogenerated style.